code/modules/clothing/spacesuits/rig/modules/utility.dm

```diff
@@ -418,29 +428,14 @@
 	interface_name = "mop projector"
```

Comment:
How is this mop different from the mop in `mounted_guns.dm`?

Reply:
These are the de-selected files.